### PR TITLE
add mspider-traffic-governance-gateway images

### PR DESCRIPTION
**Pull request by Fan-Lin**
related to #3264

### CODE REVIEW

File: docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md [hunk 1]
```diff
@@ -42,18 +42,18 @@ spec:
 
 Service mesh provides two creation methods: wizard and YAML. The specific steps to create through the wizard are as follows:
 
-1. In the left navigation bar, click `Traffic Governance` -> `Gateway Rules`, and click the `Create` button in the upper right corner.
+1. In the left navigation bar, click `Traffic Governance` -> `Gateway`, and click the `Create` button in the upper right corner.
 
-    
+    ![创建](../images/gateway01.png)
```
Comment (windsonsea) [inline docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md:47]:
```suggestion
    ![click create](../images/gateway01.png)
```
alt text 也要是英语

File: docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md [hunk 2]
```diff
@@ -42,18 +42,18 @@ spec:
...
 
-2. In the `Create Gateway Rule` interface, configure the basic information, add the server as needed, and click `OK`.
+2. In the `Create Gateway` interface, configure the basic information, add the server as needed, and click `OK`.
 
-    
+    ![创建](../images/gateway02.png)
```
Comment (windsonsea) [inline docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md:51]:
```suggestion
    ![basic info](../images/gateway02.png)
```

File: docs/zh/docs/mspider/user-guide/traffic-governance/gateway-rules.md [hunk 3]
```diff
@@ -44,16 +44,16 @@ spec:
 
 1. 在左侧导航栏点击`流量治理` -> `网关规则`，点击右上角的`创建`按钮。
 
-    ![创建](https://docs.daocloud.io/daocloud-docs-images/docs/mspider/images/gaterule01.png)
+    ![创建](../images/gateway01.png)
 
 2. 在`创建网关规则`界面中，配置基本信息，并根据需要添加服务端后点击`确定`。
 
-    ![创建网关规则](https://docs.daocloud.io/daocloud-docs-images/docs/mspider/images/gaterule02.png)
+    ![创建网关规则](../images/gaterway02.png)
```
Comment (samzong) [inline docs/zh/docs/mspider/user-guide/traffic-governance/gateway-rules.md:51]:
```suggestion
    ![创建网关规则](../images/gateway02.png)
```
There are some issues with image referencing.

File: docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md [hunk 4]
```diff
@@ -42,18 +42,18 @@ spec:
 
 Service mesh provides two creation methods: wizard and YAML. The specific steps to create through the wizard are as follows:
 
-1. In the left navigation bar, click `Traffic Governance` -> `Gateway Rules`, and click the `Create` button in the upper right corner.
+1. In the left navigation bar, click `Traffic Governance` -> `Gateway`, and click the `Create` button in the upper right corner.
```
Comment (samzong) [inline docs/en/docs/mspider/user-guide/traffic-governance/gateway-rules.md:45]:
```suggestion
1. In the left navigation bar, click `Traffic Management` -> `Gateway`, and click the `Create` button in the upper right corner.
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**